modules/tbl/fsw/src/cfe_tbl_task_cmds.c

```diff
@@ -32,6 +32,7 @@
 */
 #include "cfe_tbl_module_all.h"
 #include "cfe_version.h"
+#include "cfe_es_api_typedefs.h"
```

Comment:
This inclusion shouldn't be necessary, because "cfe_es.h" should pull in everything needed to call the function.